### DISCUSSION TITLE
Ejs_Core Gba ( game boy advance ) freezes at 97% And replacing  loading bar with custom text ?

### DISCUSSION THREAD

**Discussion by adeniyisikirat3** [EmulatorJS/EmulatorJS#470] [Q&A]
What causes the EJS_core = 'gba' game loading to sometimes freeze forever at   ' Decompress Game Core 97% ' ,  as seen in this screenshot -> https://postimg.cc/LgmDbDDV

How can I avoid this freezing? Cause it's the same problem for the GBA section on : https://www.emulatorjs.com/gba.html ( I know you're not affiliated with them. )

,

Also how can I replace the loading text / bar as seen in this screenshot ->  https://postimg.cc/FdVg3cvk

How do I replace it with a customized text for example a text like:  " please wait a few minutes while game loads " ?

**Comment by ethanaobrien** [comment]
>What causes the EJS_core = 'gba' game loading to sometimes freeze forever at   ' Decompress Game Core 97% ' 

What browser are you using? You are using the new cores, correct? What is your OS?

>as seen in this screenshot -> https://postimg.cc/LgmDbDDV

I do not open external images (because of tracking, cookies, popups, ads, etc.), you can upload images straight to github if you would like for me to view them

Example:
![Screenshot 2023-01-14 10 36 13 PM](https://user-images.githubusercontent.com/77750390/212523115-a71de152-6bc9-4b83-a32f-7fb7ffed2dbc.png)

>How do I replace it with a customized text for example a text like:  " please wait a few minutes while game loads " ?

You can do this by editing the code in `emu-main.js` and then re-minifying it, would you like me to provide you with where in the code this is located. The instructions to minify are located here: https://emulatorjs.org/docs4devs/Minifying.html

**Comment by adeniyisikirat3** [comment]
![IMG_20230115_044457_410](https://user-images.githubusercontent.com/78237454/212534897-555b7b92-2f8f-456a-b23e-7409677a8e24.jpg) [<- this is photo 1]

+

![IMG_20230115_044503_677](https://user-images.githubusercontent.com/78237454/212534937-ec400d70-0b42-4655-b7a1-d909ab0675ee.jpg) [<- this is photo 2]

,
Aha!! Thanks for tips on how upload works on this here github ! For some reason my mind always ignored that.
,
1
Your question: //What browser are you using? You are using the new cores, correct? What is your OS?//

Reply: since GBA was/is a handheld console. All my testings & settings are done on a mobile Android smartphone. It's what I'm the most comfortable with. Occasionally I switch between chrome & opera browser. But most of the time on chrome ( 
& Even this comment was sent with chrome browser  🙋🏾‍♀ HIYA!! )

,
2
Your instructions: //You can do this by editing the code in emu-main.js and then re-minifying it, would you like me to provide you with where in the code this is located.//

My Response: I guess they are located 
Here  > https://github.com/EmulatorJS/EmulatorJS/tree/main/data/minify

Or

Here > https://github.com/EmulatorJS/EmulatorJS/blob/main/data/emu-main.js

 right ?

The embarrassing thing is:  I have limited experience with only PHP & CSS , (html)  -   I totally suck @ JS

I tried uploading the project file of Emulator Js to my web host file manager. But the emulator was stuck at "Loading..." Forever. it didn't even get to " Decompress Game Core 97% "

The only thing that worked was creating through this -> https://emulatorjs.org/editor.html 

Is there a parameter I could add to replace with custom text? Like the ones here maybe? -> : https://emulatorjs.org/docs/Options.html
,

About new cores / old cores
And EJS_oldCores are not enabled as I didn't add that parameter.  & I think the default is set to use the new cores. So yes I think I'm Using the new cores.

**Reply by ethanaobrien** [reply]
>Is there a parameter I could add to replace with custom text?

I'll add this
